blik/src/main/java/com/adyen/checkout/blik/NewBlikDelegate.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 1/7/2022.
+ */
+
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 1/7/2022.
+ */
+
+package com.adyen.checkout.blik
+
+import com.adyen.checkout.components.PaymentComponentState
+import com.adyen.checkout.components.model.paymentmethods.PaymentMethod
+import com.adyen.checkout.components.model.payments.request.BlikPaymentMethod
+import com.adyen.checkout.components.model.payments.request.PaymentComponentData
+import com.adyen.checkout.components.util.PaymentMethodTypes
+import com.adyen.checkout.core.log.LogUtil
+import com.adyen.checkout.core.log.Logger
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+
+class NewBlikDelegate(val paymentMethod: PaymentMethod) : BlikDelegate {
```

Comment:
Also I prefer a name like `DefaultBlikDelegate`, so we don't have to rename this class when we remove the old delegate. WDYT?

Reply:
Fixed